src/user/supla_esp_devconn.c

```diff
@@ -1084,37 +1084,17 @@ supla_esp_channel_set_value(TSD_SuplaChannelNewValue *new_value) {
 				if ( ot < 0 )
 					ot = 0;
 
-				if ( ct != supla_esp_cfg.Time2[a]
-					 || ot != supla_esp_cfg.Time1[a] ) {
-
-					supla_esp_cfg.Time2[a] = ct;
-					supla_esp_cfg.Time1[a] = ot;
-
-					supla_esp_state.rs_position[a] = 0;
-					supla_esp_save_state(0);
-					supla_esp_cfg_save(&supla_esp_cfg);
-
-					//supla_log(LOG_DEBUG, "Reset RS[%i] position", a);
-
-				}
-
-				//supla_log(LOG_DEBUG, "V=%i", v);
+        supla_esp_gpio_rs_apply_new_times(a, ct, ot);
 
 				if ( v >= 10 && v <= 110 ) {
-
 					supla_esp_gpio_rs_add_task(a, v-10);
-
-				} else {
-
-					if ( v == 1 ) {
-						supla_esp_gpio_rs_set_relay(&supla_rs_cfg[a], RS_RELAY_DOWN, 1, 1);
-					} else if ( v == 2 ) {
-						supla_esp_gpio_rs_set_relay(&supla_rs_cfg[a], RS_RELAY_UP, 1, 1);
-					} else {
-						supla_esp_gpio_rs_set_relay(&supla_rs_cfg[a], RS_RELAY_OFF, 1, 1);
-					}
-
-				}
+        } else if ( v == 1 ) {
+          supla_esp_gpio_rs_set_relay(&supla_rs_cfg[a], RS_RELAY_DOWN, 1, 0);
```

Comment:
Why did you disable stop_delay here?

Reply:
It introduced 500 ms delay, which could be considered as a minium possible time with enabled relay via this interface.
Without this additional delay, it is possible to get 100-200 ms time of enable relay here.
Roller shutter "feels" more responsive to control from application when it stops immidiately after releasing.

For RS_RELAY_DOWN and UP, it was also introducing another delay - it caused that previous RS movement had to be at least 500 ms long, then there was 1s of START_DELAY.

BTW. those START/STOP delay were not working correctly when RS was controled from app buttons. In one of previous "rs bug fix" there was correction for that and RS started to respect START and STOP delays properly. 

Last, for facede blinds we need more precise control of how long relay is enabled. Forcing 500 ms on that interface may be problematic for tilting.